pillar.example

```diff
@@ -76,27 +76,33 @@ postgresql:
 
 
   # These users will be created and added to pg_hba.conf
-  # NOTE: If you create multiple users with access to the same database, the first user listed
+  # NOTE: If you create multiple users with access to the same database, the last user listed
   #       here will become the owner
   users:
-    - username: deploy
+    - username: user_with_password
       type: host
       database: production
       address: 10.1.2.0/24
       method: trust
-    - username: user_with_password
+      login: true
+    - username: example_role
       type: host
       database: production
-      address: 10.1.2.0/24
+      address: 0.0.0.0/24
```

Comment:
`0.0.0.0/24` makes no sense? I think we should continue using some example address here. Or was there a special reason you've changed it?

Reply:
Ehm, just for testing...I will change it back